docker-compose.yaml

```diff
@@ -15,7 +15,6 @@ services:
       - LAKEFS_BLOCKSTORE_S3_CREDENTIALS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
       - LAKEFS_BLOCKSTORE_S3_CREDENTIALS_ACCESS_SECRET_KEY=${AWS_SECRET_ACCESS_KEY}
       - LAKEFS_LOGGING_LEVEL
-      - LAKEFS_BLOCKSTORE_GS_CREDENTIALS_JSON
```

Comment:
just making sure we miss a merge with master and we are not taking this one out

Reply:
Why do we need this in the global `docker-compose`? It can be added by the user if GCS is needed, but our quickstart refers to local/S3 storage.